packages/react-mongo/types/react-mongo.d.ts

```diff
@@ -1,5 +1,10 @@
 import { Meteor } from 'meteor/meteor';
 import { Mongo } from 'meteor/mongo';
 import { DependencyList } from 'react';
-export declare const useSubscription: (factory: () => Meteor.SubscriptionHandle, deps?: DependencyList) => Meteor.SubscriptionHandle;
+declare type UseSubscriptionOptions = {
+    deps?: DependencyList;
+    updateOnReady?: boolean;
+};
+export declare const useSubscription: (factory: () => Meteor.SubscriptionHandle | void, deps?: DependencyList | UseSubscriptionOptions) => void;
 export declare const useCursor: <T = any>(factory: () => Mongo.Cursor<T>, deps?: DependencyList) => Mongo.Cursor<T>;
+export {};
```

Comment:
Was this added un purpose? 

Question 2: do we even need this file, when the `react-mongo.ts` already has Typescript definitions?